packages/dialtone-vue2/recipes/leftbar/contact_centers_row/contact_centers_row.vue

```diff
@@ -37,7 +37,10 @@
           </dt-emoji-text-wrapper>
         </div>
       </a>
-      <div class="dt-leftbar-row__omega">
+      <div
+        v-if="!hideActions"
+        class="dt-leftbar-row__omega"
+      >
```

Comment:
Hmm kinda weird it seems to break truncation if you disable it and then re-enable it again

![Screenshot 2024-08-13 at 11 49 26 AM](https://github.com/user-attachments/assets/6dde1b71-e52d-4328-9def-64e5f642eadb)

Maybe it will work if we change v-if to v-show instead?

Reply:
Yeah, good catch I was seeing the same thing. I don't believe the prop will change a lot during the user session. But, we have this:

```js
  watch: {
    $props: {
      deep: true,
      handler () {
        this.adjustLabelWidth();
      },
    },
  },
```

I don't think that's working as expected, or maybe it's re-adjusting before the `omega` container is added to the DOM. Any idea would be appreciated.

Reply:
This is how it's being used in the General Row:

```js
  watch: {
    $props: {
      immediate: true,
      deep: true,
      async handler () {
        this.validateProps();
        await this.$nextTick();
        this.adjustLabelWidth();
      },
    },
  },
```

Reply:
> This is how it's being used in the General Row:

Ah yeah, I bet it will fix it if you put a nextTick before the adjustLabelWidth, like it is there.

Reply:
Well apparently that's the fix. Ready to another round of review.